dspace-api/src/test/java/org/dspace/content/packager/DSpaceAIPIntegrationTest.java

```diff
@@ -0,0 +1,1001 @@
+/**
```

Comment:
Since this is an integration test, should it not be named IT* or *IT rather than *Test?  That way it will be run separately from unit tests, which allows us to have Maven cleanly do setup and teardown of the IT environment.